config/locales/devise.en.yml

```diff
@@ -0,0 +1,66 @@
+# Additional translations at https://github.com/heartcombo/devise/wiki/I18n
```

Comment:
I think this should work without this file. Delete, restart and try. Revert if not.

Reply:
allright